src/tests/nativeaot/SmokeTests/Preinitialization/Preinitialization.cs

```diff
@@ -55,7 +55,9 @@ private static int Main()
         TestReadOnlySpan.Run();
         TestStaticInterfaceMethod.Run();
         TestConstrainedCall.Run();
+#if !CODEGEN_WASM
         TestTypeHandles.Run();
```

Comment:
```suggestion
        // TODO-LLVM: enable frozen type preinit.
        TestTypeHandles.Run();
```
Something to add to the issue about this.